routes/quiz.js

```diff
@@ -73,6 +73,9 @@ router.get('/results', function(req, res) {
     else {
       modified_score = quiz_score / 3
     }
+    mod_scores.push(modified_score); // add this quiz's score to the all scores array
```

Comment:
in the fetchQuizScores. in the background on the quiz page, I pull all the old scores from mongo. Then, once I get the result from this quiz, I append it in, sort it, and call the percentile fxn on it